src/components/RiskComponents.vue

```diff
@@ -1,84 +1,161 @@
+    <!-- displays on medium and up breakpoints -->
+    <v-row class="hidden-sm-and-down">
+      <v-col
+        v-for="(risk, index) in riskData"
+        :key="risk.type"
+        cols="3"
+        :class="index && 'left-border'"
+      >
+        <RiskComponent v-bind="risk" :class="risk.type">
+          <template #icon>
+            <component
+              :is="icons[risk.type]"
+              class="component-icon"
+              :class="`${risk.riskClass}`"
+            />
+          </template>
+          <template #notes>
+            <Markdown :source="risk.notes" class="notes desktop-notes" />
+          </template>
+        </RiskComponent>
       </v-col>
     </v-row>
-    <v-row no-gutters v-else>
-      <v-col cols="12" sm-and-down>
+    <!-- displays on small and down breakpoints -->
+    <v-row no-gutters class="hidden-md-and-up">
+      <v-col cols="12">
         <v-expansion-panels flat>
           <RiskComponentDropdown
-            :activity="activity"
-            type="crowding"
-          ></RiskComponentDropdown>
-          <RiskComponentDropdown
-            :activity="activity"
-            type="droplets"
-          ></RiskComponentDropdown>
-          <RiskComponentDropdown
-            :activity="activity"
-            type="exposureTime"
-          ></RiskComponentDropdown>
-          <RiskComponentDropdown
-            :activity="activity"
-            type="ventilation"
-          ></RiskComponentDropdown>
+            v-for="(risk, index) in riskData"
+            :key="risk.type"
+            :class="index && 'top-border'"
+            v-bind="risk"
+          >
+            <template #icon>
+              <component
+                :is="icons[risk.type]"
+                class="component-icon dropdown-icon"
+                :class="`${risk.riskClass}`"
+              />
+            </template>
+            <template #notes>
+              <Markdown :source="risk.notes" class="notes dropdown-notes" />
+            </template>
+          </RiskComponentDropdown>
         </v-expansion-panels>
       </v-col>
     </v-row>
   </v-container>
 </template>
 
 <script>
+// getters
+import { mapGetters } from "vuex";
+// components
 import RiskComponent from "@/components/RiskComponent.vue";
 import RiskComponentDropdown from "@/components/RiskComponentDropdown.vue";
+// icons
+import CrowdingIcon from "@/assets/Risk_Crowding-filled.svg";
+import DropletsIcon from "@/assets/Risk_Droplets-filled.svg";
+import TimeIcon from "@/assets/Risk_Time-filled.svg";
+import VentIcon from "@/assets/Risk_Ventilation-filled.svg";
+// markdown
+import Markdown from "vue-markdown";
 
 export default {
   components: {
     RiskComponent,
-    RiskComponentDropdown
+    RiskComponentDropdown,
+    Markdown
   },
   props: {
     activity: Object
   },
+  data: () => ({
+    riskTypes: ["crowding", "droplets", "exposureTime", "ventilation"],
+    riskLabels: {
+      "1": "Low",
+      "2": "Medium"
```

Comment:
Even though it's the default, I think you should put High in explicitly.
```suggestion
      "2": "Medium",
      "3": "High"
```